galaxy/templates/configmap-scripts.yaml

```diff
@@ -0,0 +1,25 @@
+{{- range $key, $entry := .Values.initScripts -}}
+{{- if (not (eq $key "mountPath")) -}}
+{{- if $entry }}
+apiVersion: v1
+metadata:
+  name: {{ include "galaxy.fullname" $ }}-{{ $key | lower }}
```

Comment:
I was thinking that we could use one configMap for all init containers, instead of one configMap per script, just to reduce the number of artefacts?

Reply:
I'm assuming you mean `initScripts` and that would work if they're all secrets or all configmaps, but right now there is the option to set `useSecret` to true on one but not all, which requires each of them being a separate object. The only way I can think of to avoid this is maybe loop through and set a boolean to true the first time a secret/configmap is found and share that section for all subsequent secrets/configmaps, so there would be a single configmap and a single secret. i'm not sure how easy it will be to do with helm templating but I can give it a shot later on. I guess for now one question would be if we want to keep `useSecret` per script or not, because that will be an interface change if we shift later, but the implementation can be shifted without changes to users.